daemon/container.go

```diff
@@ -1140,3 +1141,15 @@ func (container *Container) copyImagePathContent(v volume.Volume, destination st
 
 	return v.Unmount()
 }
+
+func (container *Container) stopSignal() int {
+	var stopSignal syscall.Signal
+	if container.Config.StopSignal != "" {
+		stopSignal, _ = signal.ParseSignal(container.Config.StopSignal)
+	}
+
+	if int(stopSignal) == 0 {
+		stopSignal, _ = signal.ParseSignal(signal.DefaultStopSignal)
```

Comment:
🙈 `signal.ParseSignal` returns `-1` if the signal is invalid; the branch above doesn't check the error, and this branch only checks for `0`, so this can return `-1` as signal;

- https://github.com/moby/moby/pull/49471